### PR TITLE
Food database avi

### PR DESCRIPTION
**Pull request by aviralch**
Made the initial pattern matching. only matches exactly right now. 

To run python3 parse.py

### CODE REVIEW

File: api/parse.py [hunk 1]
```diff
@@ -101,8 +31,85 @@ def FoodDatabase():
...
     fruit_veg = df["fruit_veg"]
-    print(df)
-    return dairy, grain, meat, fruit_veg
+    food_groups = np.concatenate((dairy, meat, grain, fruit_veg))
+    # print(type(food_groups))
+    #return dairy, grain, meat, fruit_veg
+    return food_groups
+
+def nlp():
+    nlp = spacy.load('en_core_web_sm')
```
Comment (kevmo314) [inline api/parse.py:44]: Name this var something different from the function name, prevents confusion from thinking it's recursive.

File: api/parse.py [hunk 2]
```diff
@@ -101,8 +31,85 @@ def FoodDatabase():
...
+    matcher = PhraseMatcher(nlp.vocab, attr = 'LOWER')
+    parsed_words = parse()
+    words_string = ' '.join(parsed_words)
+    food_groups = food_database()
+    fruit_veg_string = food_groups[3]
+    #fruit_veg_string = np.array2string(food_groups[3])
+    #patterns = [nlp(text) for text in fruit_veg_string]
+    patterns = [nlp(text) for text in food_groups]
+    matcher.add("TerminologyList", None, *patterns)
```
Comment (kevmo314) [inline api/parse.py:55]: Some comments to explain what's going on here would be helpful (not specifically this line, just the various steps in this function)

**Review by kevmo314 — APPROVED** [review]
Good stuff :)

**Review by aviralch — COMMENTED** [review]
incorporated the suggestions - changing the function name and adding comments

**Pull request merged**